python/setup.py

```diff
@@ -14,20 +18,24 @@
     from setuptools import setup
     from setuptools.extension import Extension
 
+# We can not import `mxnet.info.py` in setup.py directly since mxnet/__init__.py
+# Will be invoked which introduces dependences
 CURRENT_DIR = os.path.dirname(__file__)
-libinfo_py = os.path.join(CURRENT_DIR, 'tvm/_ffi/libinfo.py')
+libinfo_py = os.path.join(CURRENT_DIR, './tvm/_ffi/libinfo.py')
 libinfo = {'__file__': libinfo_py}
-exec(compile(open(libinfo_py, 'rb').read(), libinfo_py, 'exec'), libinfo, libinfo)
+exec(compile(open(libinfo_py, "rb").read(), libinfo_py, 'exec'), libinfo, libinfo)
 
 LIB_PATH = libinfo['find_lib_path']()
-print(LIB_PATH)
 __version__ = libinfo['__version__']
 
 def config_cython():
     """Try to configure cython and return cython configuration"""
     if os.name == 'nt':
         print("WARNING: Cython is not supported on Windows, will compile without cython module")
         return []
+    if "i386" in sysconfig.get_config_var("CFLAGS"):
+        print("WARNING: Cython is not linking with correct arch, will compile without cython module")
```

Comment:
note that x64 flag need to be checked, it is not "x64", but something like "x86_64"